x/ccv/consumer/types/params.go

```diff
@@ -161,3 +174,21 @@ func validateProviderFeePoolAddrStr(i interface{}) error {
 	// Otherwise validate as usual for a bech32 address
 	return ccvtypes.ValidateBech32(i)
 }
+
+func validateSoftOptOutThreshold(i interface{}) error {
+	str, ok := i.(string)
+	if !ok {
+		return fmt.Errorf("invalid parameter type: %T", i)
+	}
+	dec, err := sdktypes.NewDecFromStr(str)
+	if err != nil {
+		return err
+	}
+	if dec.IsNegative() {
+		return fmt.Errorf("soft opt out threshold cannot be negative, got %s", str)
+	}
+	if dec.Sub(sdktypes.MustNewDecFromStr("0.2")).IsPositive() {
```

Comment:
This doesn't allow `0.2`. 
```suggestion
	if !dec.Sub(sdktypes.MustNewDecFromStr("0.2")).IsNegative() {
```